### DISCUSSION TITLE
Leave Multiple Approvals intended behaviors

### DISCUSSION THREAD

**Discussion by stevenfamy** [horilla-opensource/horilla#330] [General]
Hi Horilla team, 

We have a question regarding Multiple Approvals. What is the intended behavior that was implemented? Currently, what we observe is as follows: Employee G has a reporting manager named B. For his department, we have set up Multiple Approvals, requiring that if he applies for leave, it needs approval from three people: A, B, and C. However, in the current implementation, which we suspect may be a bug (but want to confirm first), when G applies for leave and C approves it, the status is already changed to 'Approved'. 

so for now I need to change the status manually from the database set to "requested" but after the other 2 approval the status need to manually change to "approved".

Thanks Before.

**Comment by horilla-opensource** [comment]
> Hi Horilla team,
> 
> We have a question regarding Multiple Approvals. What is the intended behavior that was implemented? Currently, what we observe is as follows: Employee G has a reporting manager named B. For his department, we have set up Multiple Approvals, requiring that if he applies for leave, it needs approval from three people: A, B, and C. However, in the current implementation, which we suspect may be a bug (but want to confirm first), when G applies for leave and C approves it, the status is already changed to 'Approved'.
> 
> so for now I need to change the status manually from the database set to "requested" but after the other 2 approval the status need to manually change to "approved".
> 
> Thanks Before.

Hi @stevenfamy ,

In Horilla, by default, when an employee requests leave, the employee's reporting manager or the super admin can approve or reject the request, and the status automatically updates based on their action.

However, some companies require multiple approvals for leave requests that exceed 2 or 3 days. In such cases, you can use the 'Multiple Approval Condition' feature in Horilla.

To set this up go to Horilla > Configuration > Multiple Approval Condition:

- **Department** : Select the department for which multiple approvals are needed.

- **Condition Field** : Choose 'Leave request days.'

- **Condition Operator**: Select the appropriate operator (e.g., greater than, equal to) and specify the number of requested days in the 'Condition Value' field.

- **Approval Hierarchy**: Set up the hierarchy by selecting the first, second, and subsequent approvers. The request will only move to the next manager if the previous one approves it.

If a super admin approves the request, all other managers in the hierarchy will automatically approve it as well.
